R/utils.R

```diff
@@ -1025,7 +1025,9 @@ select_choice <- function(choices, title = NULL, multiple = FALSE) {
     if (!multiple) title = paste(title, ask, sep = "\n")
   }
   if (!is.null(nm <- names(choices))) {
-    choices = paste0(choices, ifelse(is.na(nm) | nm == '', '', c(" ", nm)))
```

Comment:
Oh sorry for the misunderstanding 🤔 it did not work as expected when I used it so I reverted. 
Anyway, I'll look closer tomorrow to see if there is still the issue I have seen.

Reply:
It didn't work probably because the length of `c(" ", nm)` is not equal to `length(nm)`. Or perhaps you misunderstood `ifelse()`. Anyway, time for you to go to bed now! 💤